### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by dwapetra** [entropyxyz/community#129] [Get Test Funds] [answered]
### Address

5CrJBFn9UMYXzaYYiFcZFgr5Bv8g3ztBy9LYXQ2wSvP4SX46

**Comment by johnnymatthews** [comment, marked as answer]
Sent in block [613,513](https://polkadot.js.org/apps/#/explorer/query/0x22025ffadb1827f55be9d36d1cc9d6a34a0a85ed6ff72835a796b3a1e6559737)
